addons/l10n_it/data/account_tax_template.xml

```diff
@@ -144,6 +144,7 @@
         <field name="sequence">3</field>
         <field name="amount">20</field>
         <field name="amount_type">percent</field>
+        <field name="active">False</field>
```

Comment:
Making the tax inactive makes sense, but I don't see the point of making the template inactive? There shouldn't be any link to the template.